packages/client-sdk-nodejs/src/config/configurations.ts

```diff
@@ -18,7 +18,7 @@ const defaultLoggerFactory: MomentoLoggerFactory =
   new DefaultMomentoLoggerFactory();
 const defaultMiddlewares: Middleware[] = [];
 
-function defaultRetryStrategy(
```

Comment:
it is for now because I had to create a CacheClientConfiguration to pass to the ControlClient, until I spend the time to separate those two control clients.

Do you have concerns about this being exported?

Reply:
Nope. Was just asking if it was an oversight.